core/model/modx/processors/system/log/getlist.class.php

```diff
@@ -69,6 +71,8 @@ public function getData() {
         /* check filters */
         $wa = array();
         if (!empty($actionType)) { $wa['action:LIKE'] = '%'.$actionType.'%'; }
+        if (!empty($classKey)) { $wa['classKey:LIKE'] = '%'.$classKey.'%'; }
```

Comment:
What Jako said, also just keeping with the same search standards of the other fields